ml-agents/mlagents/trainers/ppo/multi_gpu_policy.py

```diff
@@ -43,7 +43,7 @@ def create_model(
         self.devices = get_devices()
         self.towers = []
         with self.graph.as_default():
-            with tf.variable_scope(TOWER_SCOPE_NAME, reuse=tf.AUTO_REUSE):
```

Comment:
Why remove TOWER_SCOPE_NAME here?

Reply:
This reminds me I need to remove it above as well.  It's because this scope was unnecessary (and applied to everything) -- and it also broke the names of the `output_nodes` which were used for saving the graph / inference compatibility.

Reply:
Do you still plan on removing it ?

Reply:
Yes -- still needed for compatibility of the `.nn` file